package.json

```diff
@@ -60,6 +60,7 @@
     "lodash.merge": "^4.6.1",
     "lodash.snakecase": "^4.1.1",
     "merge-stream": "^1.0.1",
+    "p-limit": "^2.0.0",
```

Comment:
You probably want to add `@types/p-limit` as well?

Reply:
![image](https://user-images.githubusercontent.com/36055624/49806584-197b0780-fd7e-11e8-812c-04313da27bf9.png)
I face this error on adding "@type/p-limit" 
Not sure if I should add this...

Reply:
Just means you need to import it using `import pLimit = require('p-limit');` which is admittedly confusing.